sklearn/covariance/tests/test_covariance.py

```diff
@@ -245,6 +245,32 @@ def test_ledoit_wolf_large():
     assert_almost_equal(lw.covariance_, cov)
 
 
+def test_ledoit_wolf_empty_array():
+    # test that ledoit wolf raises
+    # a ValueError when providing an empty array
+    X_empty = np.zeros((0, 2))
+    lw = LedoitWolf()
+    with pytest.raises(ValueError, match="Found array with 0 sample"):
+        lw.fit(X_empty)
+
+
+def test_ledoit_wolf_shrinkage_empty_array():
+    # test that ledoit wolf shrinkage raises
+    # a ValueError when providing an empty array
+    X_empty = np.zeros((0, 2))
+    with pytest.raises(ValueError, match="Found array with 0 sample"):
+        ledoit_wolf_shrinkage(X_empty)
+
+
+def test_ledoit_wolf_object_empty_array():
+    # test that LedoitWolf method `fit` raises
+    # a ValueError when providing an empty array
+    X_empty = np.zeros((0, 2))
+    lw = LedoitWolf()
+    with pytest.raises(ValueError, match="Found array with 0 sample"):
+        lw.fit(X_empty)
```

Comment:
We already have a test in `test_common.py` that checks this for all estimators in scikit-learn so you can delete the `test_ledoit_wolf_object_empty_array` test from this PR.